code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -173,7 +173,7 @@
 	icon_state = "smg9mm-42"
 	ammo_type = /obj/item/ammo_casing/c9mm
 	caliber = "9mm"
-	max_ammo = 42
+	max_ammo = 30

```

Comment:
since you change the max ammo to 30 you might want to rename the icon_state to "smg9mm-30"